### PR TITLE
Mb/load data into patient objs

### PR DESCRIPTION
**Pull request by MartinBernstorff**
A proof-of-conccept for how to load from the SQL and unpack into patient objects. Please review all code in 
- psycop/common/feature_generation/sequences/mlm_sequence_loaders/MLMLoaders.py
- psycop/common/feature_generation/sequences/mlm_sequence_loaders/test_diagnosis_loader.py

I've tested it on Ovartaci, and it takes about 90s.

Apologies about the weird PR; merged in by mistake, and now it'll be pretty hard to undo the merges on main.

@KennethEnevoldsen, @sarakolding, would love reviews from both of you 👍 

After this, we will restructure dirs according to #208. 

(To hide the test-coverage annotations, you press `A`)

### CODE REVIEW

File: psycop/common/feature_generation/sequences/mlm_sequence_loaders/MLMLoaders.py [hunk 1]
```diff
@@ -55,12 +75,21 @@ class MLMDataLoader(AbstractMLMDataLoader):
...
-        ).unpack(source_event_dataframes=[events_from_train.collect()])
+        ).unpack(source_event_dataframes=[events_after_2013.collect()])
 
         return unpacked_patients
+
+
+if __name__ == "__main__":
+    patients = MLMDataLoader.get_train_set(loaders=[DiagnosisLoader()])
+
+    pass
```
Comment (sarakolding) [inline psycop/common/feature_generation/sequences/mlm_sequence_loaders/MLMLoaders.py:95]: pass?

File: psycop/common/feature_generation/sequences/mlm_sequence_loaders/MLMLoaders.py [hunk 2]
```diff
@@ -55,12 +75,21 @@ class MLMDataLoader(AbstractMLMDataLoader):
     @staticmethod
```
Comment (sarakolding) [inline psycop/common/feature_generation/sequences/mlm_sequence_loaders/MLMLoaders.py:75]: bit confused about the naming here... ask me why

File: psycop/common/feature_generation/sequences/mlm_sequence_loaders/test_diagnosis_loader.py [hunk 3]
```diff
@@ -15,4 +15,4 @@ def test_a_diagnosis_formatting():
     formatted_df = DiagnosisLoader().format_diagnosis_columns(df=df.lazy())
 
     resulting_diagnoses = formatted_df.collect().get_column("value").to_list()
-    assert resulting_diagnoses == ["DF431", "DF431", "ALFC3", "DF329"]
+    assert resulting_diagnoses == ["F431", "F431", "ALFC3", "F329"]
```
Comment (sarakolding) [inline psycop/common/feature_generation/sequences/mlm_sequence_loaders/test_diagnosis_loader.py:18]: should we also test that the type gets parsed correctly?

File: psycop/common/feature_generation/sequences/mlm_sequence_loaders/MLMLoaders.py [hunk 4]
```diff
@@ -28,39 +26,61 @@ class DiagnosisLoader(EventDfLoader):
...
         unpacked_patients = EventDataFramesToPatients(
             column_names=PatientColumnNames(),
-        ).unpack(source_event_dataframes=[events_from_train.collect()])
+        ).unpack(source_event_dataframes=[events_after_2013.collect()])
 
         return unpacked_patients
+
+
+if __name__ == "__main__":
+    patients = MLMDataLoader.get_train_set(loaders=[DiagnosisLoader()])
```
Comment (KennethEnevoldsen) [inline psycop/common/feature_generation/sequences/mlm_sequence_loaders/MLMLoaders.py:86]: I wouldn't call it MLM as it does not do any of the masking. I would call it PatientLoader

The problem with using the dataloader is that it might be confused with the pytorch dataloader.
